elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/queryengines/sql/metadata/SQLReferenceVisitor.java

```diff
@@ -69,7 +69,7 @@ protected String visitFieldDimension(ColumnProjection dimension) {
 
         //This is a nested query.  Don't do table lookups.
         } else {
-            return getFieldAlias(applyQuotes(tableAliases.peek()), applyQuotes(dimension.getName()));
+            return getFieldAlias(applyQuotes(tableAliases.peek()), applyQuotes(dimension.getSQLAlias()));
```

Comment:
What about metrics?

Reply:
this method gets dimensions when columnType is `FIELD` (https://github.com/yahoo/elide/blob/master/elide-datastore/elide-datastore-aggregation/src/main/java/com/yahoo/elide/datastores/aggregation/metadata/ColumnVisitor.java#L59)
Metric will  goes to visitFormulaMetric --> visitFormulaColumn --> visitPhysicalReference